### DISCUSSION TITLE
./build.sh Unable to load AWS_CREDENTIAL_FILE

### DISCUSSION THREAD

**Discussion by modevico** [google/orbit#4439] [Q&A]
Trying to install/build orbit from main branch on RHEL 8.5, but am getting the following error:

Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
Failure: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727).

$ ./build.sh

abseil/20211102.0: Package is up to date
bzip2/1.0.8: Package is up to date
c-ares/1.15.0: Package is up to date
capstone/4.0.2: Package is up to date
glad/0.1.34: Package is up to date
imgui/1.85: Package is up to date
libunwindstack-android-dependencies/20210709@orbitdeps/stable: Package is up to date
lzma_sdk/19.00@orbitdeps/stable: Package is up to date
openssl/1.1.1k: Package is up to date
outcome/2.2.0: Package is up to date
protobuf/3.9.1@bincrafters/stable: Package is up to date
vulkan-headers/1.1.114.0: Package is up to date
zlib/1.2.11: Package is up to date
crashpad/20200624@orbitdeps/stable: WARN: Build-require 'depot_tools' cannot be found in lockfile
crashpad/20200624@orbitdeps/stable: WARN: Build-require 'ninja' cannot be found in lockfile
depot_tools/20200407: Package is up to date
ninja/1.10.2: Package is up to date
protoc_installer/3.9.1@bincrafters/stable: Package is up to date
grpc_codegen/1.27.3@orbitdeps/stable: Package is up to date
libprotobuf-mutator/20200506@orbitdeps/stable: WARN: Build-require 'protoc_installer' cannot be found in lockfile
protobuf/3.9.1@bincrafters/stable: Package is up to date
llvm-core/12.0.0@orbitdeps/stable: WARN: Build-require 'cmake' cannot be found in lockfile
openssl/1.1.1k: Package is up to date
cmake/3.20.3: Package is up to date
gtest/1.11.0: Package is up to date
conanfile.py (OrbitProfiler/None@orbitdeps/stable): WARN: Build-require 'cmake_installer' cannot be found in lockfile
cmake_installer/3.16.3@conan/stable: WARN: [DEPRECATED] Package cmake_installer/3.16.3@conan/stable is being deprecated. Change yours to require cmake/3.16.3 instead
cmake_installer/3.16.3@conan/stable: Package is up to date
Requirements
    abseil/20211102.0 from 'bintray' - Cache
    bzip2/1.0.8 from 'bintray' - Cache
    c-ares/1.15.0 from 'bintray' - Cache
    capstone/4.0.2 from 'bintray' - Cache
    crashpad/20200624@orbitdeps/stable from 'bintray' - Cache
    freetype/2.10.0@bincrafters/stable from 'bintray' - Cache
    freetype-gl/79b03d9@orbitdeps/stable from 'bintray' - Cache
    glad/0.1.34 from 'bintray' - Cache
    grpc/1.27.3@orbitdeps/stable from 'bintray' - Cache
    imgui/1.85 from 'bintray' - Cache
    libpng/1.6.37@bincrafters/stable from 'bintray' - Cache
    libprotobuf-mutator/20200506@orbitdeps/stable from 'bintray' - Cache
    libssh2/1.9.0 from 'bintray' - Cache
    libunwindstack-android-dependencies/20210709@orbitdeps/stable from 'bintray' - Cache
    llvm-core/12.0.0@orbitdeps/stable from 'bintray' - Cache
    lzma_sdk/19.00@orbitdeps/stable from 'bintray' - Cache
    openssl/1.1.1k from 'bintray' - Cache
    outcome/2.2.0 from 'conan-center' - Cache
    protobuf/3.9.1@bincrafters/stable from 'bintray' - Cache
    volk/1.2.170 from 'bintray' - Cache
    vulkan-headers/1.1.114.0 from 'conan-center' - Cache
    zlib/1.2.11 from 'bintray' - Cache
Packages
    abseil/20211102.0:6eeea3904f53d9f7311f5a5b447089286767e0b4 - Cache
    bzip2/1.0.8:f0542bfee356339312487ee9552f7a811e256533 - Cache
    c-ares/1.15.0:d8495d3c468ddc45cc8454e690af02a181a8f60c - Cache
    capstone/4.0.2:9f4bcceafc864d01718e155f629f2e878618725a - Cache
    crashpad/20200624@orbitdeps/stable:c7f7792208a14227efdf8360c42b28462d7faba0 - Build
    freetype/2.10.0@bincrafters/stable:d2a93b5ccb49910ad11c11d3d5564a766a2264f7 - Build
    freetype-gl/79b03d9@orbitdeps/stable:582b6bfaf2dab3a74e87c292ff362199be06a24a - Build
    glad/0.1.34:6729372a2aec44a8b534ba829bbced3d963bc8b1 - Cache
    grpc/1.27.3@orbitdeps/stable:5656350a0817b391e6aeebae0cd8ddf0497a5454 - Build
    imgui/1.85:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    libpng/1.6.37@bincrafters/stable:b964726cd5abb10334ecc2e032bb1ae533ce3e6a - Build
    libprotobuf-mutator/20200506@orbitdeps/stable:16655cbc4c3d8d5c9975b4ee4e60e8c65a1f4447 - Build
    libssh2/1.9.0:c2585668acfa4c89817523237b4cf8834fa6ded3 - Build
    libunwindstack-android-dependencies/20210709@orbitdeps/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    llvm-core/12.0.0@orbitdeps/stable:21fb15017eee317f44330b7f5fbd7b82e6b15c29 - Build
    lzma_sdk/19.00@orbitdeps/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    openssl/1.1.1k:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
    outcome/2.2.0:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    protobuf/3.9.1@bincrafters/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    volk/1.2.170:6b4b18b78ab8e483b1828b9e0fa14284b2a7ad37 - Build
    vulkan-headers/1.1.114.0:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    zlib/1.2.11:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
Build requirements
    cmake/3.20.3 from 'conan-center' - Cache
    cmake_installer/3.16.3@conan/stable from 'bintray' - Cache
    depot_tools/20200407 from 'conan-center' - Cache
    grpc_codegen/1.27.3@orbitdeps/stable from 'bintray' - Cache
    gtest/1.11.0 from 'bintray' - Cache
    ninja/1.10.2 from 'conan-center' - Cache
    openssl/1.1.1k from 'bintray' - Cache
    protobuf/3.9.1@bincrafters/stable from 'bintray' - Cache
    protoc_installer/3.9.1@bincrafters/stable from 'bintray' - Cache
Build requirements packages
    cmake/3.20.3:5c09c752508b674ca5cb1f2d327b5a2d582866c8 - Cache
    cmake_installer/3.16.3@conan/stable:44fcf6b9a7fb86b2586303e3db40189d3b511830 - Cache
    depot_tools/20200407:cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31 - Cache
    grpc_codegen/1.27.3@orbitdeps/stable:b8d284b105831af7e74cfe6ff698a2a0c917cac3 - Cache
    gtest/1.11.0:addf869846356ee78f18e4d283040b3ff888dbfc - Cache
    ninja/1.10.2:252f7e09f641bef29a75fc98d05ac217b842cde4 - Cache
    openssl/1.1.1k:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
    protobuf/3.9.1@bincrafters/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Skip
    protoc_installer/3.9.1@bincrafters/stable:c0c1ef10e3d0ded44179e28b669d6aed0277ca6a - Cache

Generated lockfile: /local/mnt/workspace/mdevico/repo/orbit/build_default_relwithdebinfo/conan.lock
Using lockfile: '/local/mnt/workspace/mdevico/repo/orbit/build_default_relwithdebinfo/conan.lock'
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=RelWithDebInfo
compiler=gcc
compiler.fpo=False
compiler.libcxx=libstdc++11
compiler.version=9
os=Linux
os_build=Linux
cmake_installer:build_type=Release
cmake:build_type=Release
cmake:os=Linux
cmake:os.platform=None
cmake:os_build=Linux
cmake:arch=x86_64
cmake:arch_build=x86_64
abseil:compiler=gcc
abseil:compiler.cppstd=17
gtest:compiler=gcc
gtest:compiler.cppstd=17
[options]
[build_requires]
&: cmake_installer/3.16.3@conan/stable
[env]
CC=gcc-9
CFLAGS=-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -D_FORTIFY_SOURCE=2 -fstack-protector-all
CXX=g++-9
CXXFLAGS=-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fsized-deallocation -D_FORTIFY_SOURCE=2 -fstack-protector-all -fno-exceptions
LDFLAGS=-Wl,-z,relro,-z,now,-z,noexecstack
gtest:CXXFLAGS=-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fsized-deallocation -D_FORTIFY_SOURCE=2 -fstack-protector-all -fno-exceptions -std=c++17
abseil/20211102.0: Package is up to date
bzip2/1.0.8: Package is up to date
c-ares/1.15.0: Package is up to date
capstone/4.0.2: Package is up to date
glad/0.1.34: Package is up to date
imgui/1.85: Package is up to date
libunwindstack-android-dependencies/20210709@orbitdeps/stable: Package is up to date
lzma_sdk/19.00@orbitdeps/stable: Package is up to date
openssl/1.1.1k: Package is up to date
outcome/2.2.0: Package is up to date
protobuf/3.9.1@bincrafters/stable: Package is up to date
vulkan-headers/1.1.114.0: Package is up to date
zlib/1.2.11: Package is up to date
depot_tools/20200407: Package is up to date
ninja/1.10.2: Package is up to date
protoc_installer/3.9.1@bincrafters/stable: Package is up to date
grpc_codegen/1.27.3@orbitdeps/stable: Package is up to date
cmake/3.20.3: Package is up to date
gtest/1.11.0: Package is up to date
cmake_installer/3.16.3@conan/stable: WARN: [DEPRECATED] Package cmake_installer/3.16.3@conan/stable is being deprecated. Change yours to require cmake/3.16.3 instead
cmake_installer/3.16.3@conan/stable: Package is up to date
conanfile.py (OrbitProfiler/None): Installing package
Requirements
    abseil/20211102.0 from 'bintray' - Cache
    bzip2/1.0.8 from 'bintray' - Cache
    c-ares/1.15.0 from 'bintray' - Cache
    capstone/4.0.2 from 'bintray' - Cache
    crashpad/20200624@orbitdeps/stable from 'bintray' - Cache
    freetype/2.10.0@bincrafters/stable from 'bintray' - Cache
    freetype-gl/79b03d9@orbitdeps/stable from 'bintray' - Cache
    glad/0.1.34 from 'bintray' - Cache
    grpc/1.27.3@orbitdeps/stable from 'bintray' - Cache
    imgui/1.85 from 'bintray' - Cache
    libpng/1.6.37@bincrafters/stable from 'bintray' - Cache
    libprotobuf-mutator/20200506@orbitdeps/stable from 'bintray' - Cache
    libssh2/1.9.0 from 'bintray' - Cache
    libunwindstack-android-dependencies/20210709@orbitdeps/stable from 'bintray' - Cache
    llvm-core/12.0.0@orbitdeps/stable from 'bintray' - Cache
    lzma_sdk/19.00@orbitdeps/stable from 'bintray' - Cache
    openssl/1.1.1k from 'bintray' - Cache
    outcome/2.2.0 from 'conan-center' - Cache
    protobuf/3.9.1@bincrafters/stable from 'bintray' - Cache
    volk/1.2.170 from 'bintray' - Cache
    vulkan-headers/1.1.114.0 from 'conan-center' - Cache
    zlib/1.2.11 from 'bintray' - Cache
Packages
    abseil/20211102.0:6eeea3904f53d9f7311f5a5b447089286767e0b4 - Cache
    bzip2/1.0.8:f0542bfee356339312487ee9552f7a811e256533 - Cache
    c-ares/1.15.0:d8495d3c468ddc45cc8454e690af02a181a8f60c - Cache
    capstone/4.0.2:9f4bcceafc864d01718e155f629f2e878618725a - Cache
    crashpad/20200624@orbitdeps/stable:c7f7792208a14227efdf8360c42b28462d7faba0 - Build
    freetype/2.10.0@bincrafters/stable:d2a93b5ccb49910ad11c11d3d5564a766a2264f7 - Build
    freetype-gl/79b03d9@orbitdeps/stable:582b6bfaf2dab3a74e87c292ff362199be06a24a - Build
    glad/0.1.34:6729372a2aec44a8b534ba829bbced3d963bc8b1 - Cache
    grpc/1.27.3@orbitdeps/stable:5656350a0817b391e6aeebae0cd8ddf0497a5454 - Build
    imgui/1.85:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    libpng/1.6.37@bincrafters/stable:b964726cd5abb10334ecc2e032bb1ae533ce3e6a - Build
    libprotobuf-mutator/20200506@orbitdeps/stable:16655cbc4c3d8d5c9975b4ee4e60e8c65a1f4447 - Build
    libssh2/1.9.0:c2585668acfa4c89817523237b4cf8834fa6ded3 - Build
    libunwindstack-android-dependencies/20210709@orbitdeps/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    llvm-core/12.0.0@orbitdeps/stable:21fb15017eee317f44330b7f5fbd7b82e6b15c29 - Build
    lzma_sdk/19.00@orbitdeps/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    openssl/1.1.1k:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
    outcome/2.2.0:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    protobuf/3.9.1@bincrafters/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Cache
    volk/1.2.170:6b4b18b78ab8e483b1828b9e0fa14284b2a7ad37 - Build
    vulkan-headers/1.1.114.0:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    zlib/1.2.11:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
Build requirements
    cmake/3.20.3 from 'conan-center' - Cache
    cmake_installer/3.16.3@conan/stable from 'bintray' - Cache
    depot_tools/20200407 from 'conan-center' - Cache
    grpc_codegen/1.27.3@orbitdeps/stable from 'bintray' - Cache
    gtest/1.11.0 from 'bintray' - Cache
    ninja/1.10.2 from 'conan-center' - Cache
    openssl/1.1.1k from 'bintray' - Cache
    protobuf/3.9.1@bincrafters/stable from 'bintray' - Cache
    protoc_installer/3.9.1@bincrafters/stable from 'bintray' - Cache
Build requirements packages
    cmake/3.20.3:5c09c752508b674ca5cb1f2d327b5a2d582866c8 - Cache
    cmake_installer/3.16.3@conan/stable:44fcf6b9a7fb86b2586303e3db40189d3b511830 - Cache
    depot_tools/20200407:cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31 - Cache
    grpc_codegen/1.27.3@orbitdeps/stable:b8d284b105831af7e74cfe6ff698a2a0c917cac3 - Cache
    gtest/1.11.0:addf869846356ee78f18e4d283040b3ff888dbfc - Cache
    ninja/1.10.2:252f7e09f641bef29a75fc98d05ac217b842cde4 - Cache
    openssl/1.1.1k:5f0f055dda9c222e7a0f322f0033f71083968a38 - Cache
    protobuf/3.9.1@bincrafters/stable:22be83f2b36c420bae6cbe4852995baafd502055 - Skip
    protoc_installer/3.9.1@bincrafters/stable:c0c1ef10e3d0ded44179e28b669d6aed0277ca6a - Cache

Installing (downloading, building) binaries...
abseil/20211102.0: Already installed!
bzip2/1.0.8: Already installed!
bzip2/1.0.8: Appending PATH environment variable: /usr2/mdevico/.conan/data/bzip2/1.0.8/_/_/package/f0542bfee356339312487ee9552f7a811e256533/bin
c-ares/1.15.0: Already installed!
c-ares/1.15.0: Appending PATH environment variable: /usr2/mdevico/.conan/data/c-ares/1.15.0/_/_/package/d8495d3c468ddc45cc8454e690af02a181a8f60c/bin
capstone/4.0.2: Already installed!
cmake_installer/3.16.3@conan/stable: Already installed!
depot_tools/20200407: Already installed!
depot_tools/20200407: Appending PATH env var with : /usr2/mdevico/.conan/data/depot_tools/20200407/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin
glad/0.1.34: Already installed!
gtest/1.11.0: Already installed!
imgui/1.85: Already installed!
imgui/1.85: Appending PATH env var with : /usr2/mdevico/.conan/data/imgui/1.85/_/_/package/22be83f2b36c420bae6cbe4852995baafd502055/bin
libunwindstack-android-dependencies/20210709@orbitdeps/stable: Already installed!
lzma_sdk/19.00@orbitdeps/stable: Already installed!
ninja/1.10.2: Already installed!
openssl/1.1.1k: Already installed!
outcome/2.2.0: Already installed!
protobuf/3.9.1@bincrafters/stable: Already installed!
vulkan-headers/1.1.114.0: Already installed!
zlib/1.2.11: Already installed!
cmake/3.20.3: Already installed!
cmake/3.20.3: Appending PATH environment variable: /usr2/mdevico/.conan/data/cmake/3.20.3/_/_/package/5c09c752508b674ca5cb1f2d327b5a2d582866c8/bin
crashpad/20200624@orbitdeps/stable: Applying build-requirement: depot_tools/20200407
crashpad/20200624@orbitdeps/stable: Applying build-requirement: ninja/1.10.2
crashpad/20200624@orbitdeps/stable: WARN: Build folder is dirty, removing it: /usr2/mdevico/.conan/data/crashpad/20200624/orbitdeps/stable/build/c7f7792208a14227efdf8360c42b28462d7faba0
crashpad/20200624@orbitdeps/stable: WARN: Trying to remove corrupted source folder
crashpad/20200624@orbitdeps/stable: WARN: This can take a while for big packages
crashpad/20200624@orbitdeps/stable: Configuring sources in /usr2/mdevico/.conan/data/crashpad/20200624/orbitdeps/stable/source
Warning: Running gclient on Python 3. 
If you encounter any issues, please file a bug on crbug.com under the Infra>SDK component.
Warning: Running gclient on Python 3. 
If you encounter any issues, please file a bug on crbug.com under the Infra>SDK component.

________ running 'git -c core.deltaBaseCacheLimit=2g clone --no-checkout --progress https://github.com/chromium/crashpad.git --template=/local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source/_gclient_gittmp_crashpad_29gw8n_ /local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source/_gclient_crashpad_teman2sk' in '/local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source'
Cloning into '/local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source/_gclient_crashpad_teman2sk'...
remote: Enumerating objects: 32340, done.        
remote: Counting objects: 100% (32340/32340), done.        
remote: Compressing objects: 100% (5803/5803), done.        
remote: Total 32340 (delta 26830), reused 31769 (delta 26411), pack-reused 0        
Receiving objects: 100% (32340/32340), 13.34 MiB | 26.33 MiB/s, done.
Resolving deltas: 100% (26830/26830), done.
Syncing projects: 100% (13/13), done.                                          

________ running 'download_from_google_storage --no_resume --no_auth --bucket=chromium-clang-format --sha1_file buildtools/linux64/clang-format.sha1' in '/local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source'
Failed to fetch file gs://chromium-clang-format/942fc8b1789144b8071d3fc03ff0fcbe1cf81ac8 for buildtools/linux64/clang-format. [Err: /local/mnt/workspace/mdevico/repo/orbit/.conan/data/depot_tools/20200407/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/external_bin/gsutil/gsutil_4.28/gsutil/third_party/boto/boto/pyami/config.py:69: UserWarning: Unable to load AWS_CREDENTIAL_FILE ()
  warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
Failure: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727).
]
0> Failed to fetch file gs://chromium-clang-format/942fc8b1789144b8071d3fc03ff0fcbe1cf81ac8 for buildtools/linux64/clang-format, skipping. [Err: /local/mnt/workspace/mdevico/repo/orbit/.conan/data/depot_tools/20200407/_/_/package/cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31/bin/external_bin/gsutil/gsutil_4.28/gsutil/third_party/boto/boto/pyami/config.py:69: UserWarning: Unable to load AWS_CREDENTIAL_FILE ()
  warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
Failure: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727).
]
Downloading 1 files took 1.426330 second(s)
Error: Command 'download_from_google_storage --no_resume --no_auth --bucket=chromium-clang-format --sha1_file buildtools/linux64/clang-format.sha1' returned non-zero exit status 1 in /local/mnt/workspace/mdevico/repo/orbit/.conan/data/crashpad/20200624/orbitdeps/stable/source
ERROR: crashpad/20200624@orbitdeps/stable: Error in source() method, line 77
        self.run("gclient sync --no-history")
        ConanException: Error 2 while executing gclient sync --no-history

**Comment by florian-kuebler** [comment]
Hi, thanks for reporting that issue.

You should be to successfully build Orbit by disabling crashpad, as described in https://github.com/google/orbit/issues/3232.
